### ISSUE TITLE
Why the vis_thresh is 0.3?  Should we change it according to own datasest?

### ISSUE THREAD

**Issue by sisrfeng** [xingyizhou/CenterNet#793]
Thx!

**Comment by ShawnNew** [comment]
Hi, I met a similiar problem. I trained a model with my own dataset, and found that the final score is low so that I have to set a small vis_thresh.
Anyone knows why?

**Comment by cao-nv** [comment]
That is common practice in deep learning? 
A proper threshold should separate true and false prediction well.

**Comment by sisrfeng** [comment]
> A proper threshold should separate true and false prediction well.
If the best vis_thresh is 0.2, which can separate true and false prediction well, does such a low confidence mean that the model is not well trained?

**Comment by cao-nv** [comment]
> > A proper threshold should separate true and false predictions well.
> > If the best vis_thresh is 0.2, which can separate true and false prediction well, does such a low confidence mean that the model is not well trained?

No, the only target is an model that can distinguish between classes. If the threshold is 0.2 is ok, you can use Softmax with Temperature to scale the threshold to a higher/lower value easily without affecting the final accuracy.

**Comment by sisrfeng** [comment]
> use Softmax with Temperature to scale the threshold to a higher/lower value easily without affecting the final accuracy

https://stackoverflow.com/a/63471046/13335630
a) Sample 'hard'softmax probs : (0.01,0.01,0.98)
b) Sample 'soft' softmax probs : (0.2,0.2,0.6)

**Comment by cao-nv** [comment]
> > use Softmax with Temperature to scale the threshold to a higher/lower value easily without affecting the final accuracy
> 
> https://stackoverflow.com/a/63471046/13335630
> a) Sample 'hard'softmax probs : (0.01,0.01,0.98)
> b) Sample 'soft' softmax probs : (0.2,0.2,0.6)

your examples is for the case T>1, T < 1 will increase the highest score and increase the others
